scripts/hostcfgd

```diff
@@ -1611,7 +1664,7 @@ class HostConfigDaemon:
         self.passwcfg.load(passwh)
         self.devmetacfg.load(dev_meta)
         self.mgmtifacecfg.load(mgmt_ifc, mgmt_vrf)
-        self.syslogcfg.load(syslog)
```

Comment:
Should we remove the syslogcfg class altogether? With this change it looks like both syslogcfg and rsyslogcfg subscribes  to CFG_SYSLOG_CONFIG_TABLE_NAME and both do same task

Reply:
i cleaned it and combined both classes together